### DISCUSSION TITLE
The country "Helvetia" is better known as what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#613] [Galaxy Trivia] [answered]
*answer with the flag of the country*

**Comment by peterbaccaro** [comment]
![switzerland](https://user-images.githubusercontent.com/74666079/233800572-e6313081-70ea-4bcc-a490-a326f9801c6b.png)

The country "Helvetia" is better known as Switzerland. "Helvetia" is a Latin term used historically to refer to the territory that is now Switzerland, and it is often used in official seals, coins, and emblems of Switzerland.

**Comment by thuongtruong1009** [comment, marked as answer]
Helvetia is better known as Switzerland. Here is the flag of Switzerland:

🇨🇭
